pages/[id]/index.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
+import styled from 'styled-components';
 import Page from '../../components/page';
 import { getHackathonPaths, getHackathons } from '../../utility/firebase';
+import { COLOR } from '../../constants';
+
+const HomeHeader = styled.h1`
```

Comment:
nit but we can just call this Header if its the only header in this component

Reply:
this is a placeholder and probably won't make it to the final "home" screen.